projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss

```
@@ -103,6 +103,27 @@
   }
 }
 
+@mixin generate-dropdown-header() {
+  display: block;
+  color: dropdown-variables.$clr-dropdown-header-color;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-align: left;
+  justify-content: start;
```

Why do we force th overflow behaviour?


```suggestion
  color: dropdown-variables.$clr-dropdown-header-color;
```

---

Use `@extend %dropdown-content;`

---

We don't need titles longer than the maximum dropdown width to go on two lines.

---

[a9ab426](https://github.com/vmware-clarity/ng-clarity/pull/639/commits/a9ab426d14a1241e9f718f9f39799ee54564d0ed)